packages/blocks/src/_common/inline/presets/nodes/link-node/affine-link.ts

```diff
@@ -63,20 +63,20 @@ export class AffineLink extends ShadowlessElement {
 
   static override styles = css`
     affine-link > a {
-      white-space: nowrap;
-      word-break: break-word;
-      color: var(--affine-link-color);
-      fill: var(--affine-link-color);
-      text-decoration: none;
-      cursor: pointer;
+      white-space: nowrap !important;
```

Comment:
Is it possible avoiding using `!important`?

Reply:
https://developer.mozilla.org/en-US/docs/Web/CSS/@layer
I don't know whether this can be implemented. Just for reference.